gobblin-core/build.gradle

```diff
@@ -41,8 +41,11 @@ dependencies {
 
   if (project.hasProperty('useHadoop2')) {
     compile externalDependency.avroMapredH2
+    compile externalDependency.hadoopClientCore
+    compile externalDependency.hadoopAnnotations
   } else {
     compile externalDependency.avroMapredH1
+    compile externalDependency.hadoop
```

Comment:
Are these hadoop additions necessary? The top level build.gradle all expresses these dependencies for each subproject